src/v2/guide/list.md

```diff
@@ -440,6 +440,10 @@ De forma similar a la plantilla `v-if`, también puede usar una etiqueta `<templ
 
 ## `v-for` con `v-if`
 
+nota: Tenga en cuenta que no se recomienda usar v-if y v-for juntos. Consulte la [guía de estilo](https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential) para más detalles.
+

```

Comment:
Remember to keep a single blank line between the texts, adding the class `tip` should be displayed correctly.